src/main/java/roomescape/reservation/domain/exception/PastReservationException.java

```diff
@@ -0,0 +1,7 @@
+package roomescape.reservation.domain.exception;
+
+public class PastReservationException extends RuntimeException {
+    public PastReservationException(String message) {
+        super(message);
```

Comment:
클래스 내부에서 예외 메시지를 가지게 되면 사용처마다 문구가 미세하게 달라지는 것을 막을 수 있겠네요!

```suggestion
        super("[ERROR] 현재 시간 이후로 예약할 수 있습니다.");
```

Reply:
사용처가 여러 곳인 예외에는 클래스 내부에서 가지는 것이 좋겠네요!
그런데 해당 예외는 ReservationDateTime의 validatePast 에서만 던져질 것 같은데 또 다른 사용처가 생길 수 있을까요?
없다면 예외 메시지를 ReservationDateTime가 갖는 것이 더 가독성 있을 것 같기도 해요. 어떻게 생각하시나요?

Reply:
음 해당 방법도 좋은데요!
`PastReservationException` << 클래스명을 명확하게 잘 지어주셔서
`ReservationDateTime` 내부에서 `throw new PastReservationException()` 만 해도 가독성은 충분히 가져갈 수 있어 보여요 👍

Reply:
동의합니다! 클래스 명으로도 충분하겠네요~